.npmrc

```diff
@@ -1,3 +1,2 @@
-@types:registry=https://registry.npmjs.org/
```

Comment:
@mohanraj-r Do you remember why you needed this specific line originally?

Reply:
Not sure, guess it might have been initialized that way with npm/yarn init or one of those tools or copy[asted frpm docs